babel/src/board.js

```diff
@@ -35,6 +31,10 @@ export class Board {
         return triples.combine(this.triples);
     }
     
+    get atticOrbs() {
+        return _.cloneDeep(new Board(this.width, this.height, this.atticTypes).orbs);
+    }
```

Comment:
How would I change the types of only the attic?

`new Board().attic.orbs` would have an attic of the same types as `.obs`. Or is that too specific of a feature?